script/lint_darwin_plugins.sh

```diff
@@ -13,31 +13,52 @@ function lint_package() {
   local package_dir="${REPO_DIR}/packages/$package_name/"
   local failure_count=0
 
+  # These podspecs are temporary multi-platform adoption dummy files.
   local skipped_podspecs=(
-    'url_launcher_web.podspec'
+    "url_launcher_web.podspec"
   )
-  find "${package_dir}" -type f -name '*\.podspec' | while read podspec; do
-    # These podspecs are temporary multi-platform adoption dummy files.
-    if [[ "${skipped_podspecs[*]}" =~ "$(basename ${podspec})" ]]; then
+  
+  # TODO: These packages have analyzer warnings. Remove plugins from this list as issues are fixed.
```

Comment:
nit: Is there a bug tracking this?

Reply:
I created issues and update the comment.